### PR TITLE
add pythonPackage.wrf-python

### PR DESCRIPTION
**Pull request by mhaselsteiner**
Added python library  for post processing of WRF model output.
##### Remark
naming convention violates pep 0503 to be consistent with pypi.

### CODE REVIEW

**Comment by costrouc** [comment]
Also please change commit message to `pythonPackages.wrf-python: init at 1.2.0`

**Review by costrouc — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/wrf-python/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0maphsqfnrqpp52zz7xwcfjhw7y0fvq81mz91a9rfqvw0b2ww83m";
+  };
+  buildInputs = [ gfortran ];
+  propagatedBuildInputs = [
+            setuptools
+            wrapt
+            numpy>=1.11
```
Comment (costrouc) [inline pkgs/development/python-modules/wrf-python/default.nix]: In nix you cannot specify version. We try to make everything use the latest version. `numpy>=1.11` -> `numpy`

File: pkgs/development/python-modules/wrf-python/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            mock
+            xarray
+          ];
+
+  doCheck = false;
```
[the remark this answers is not in the document]
Comment (mhaselsteiner) [reply pkgs/development/python-modules/wrf-python/default.nix]: In the setup.py no test suite is specified and and there is also no runtests.py . Some tests are included but some dependencies on files which are not included, probably due to their size.
I will add a comment

File: pkgs/development/python-modules/wrf-python/default.nix (hunk 1 above)
Comment (mhaselsteiner) [reply pkgs/development/python-modules/wrf-python/default.nix]: Ok, thanks. I realized, when testing it again this caused my build to fail. Sorry, it's my first time contributing here..

**Comment by mhaselsteiner** [comment]
Thanks for the feedback.  I will try to apply the changes asap.

**Comment by mhaselsteiner** [comment]
I implemented all the changes. Hope it's ok now. Let me know, if there is still something not aligned with the guidelines.

File: pkgs/development/python-modules/wrf-python/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+buildPythonPackage rec {
+  pname = "wrf-python";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0maphsqfnrqpp52zz7xwcfjhw7y0fvq81mz91a9rfqvw0b2ww83m";
+  };
+  propagatedBuildInputs = [
+    gfortran
```
Comment (costrouc) [inline pkgs/development/python-modules/wrf-python/default.nix]: Is `gfortran` needed for running the package? If is only a build requirement https://github.com/NCAR/wrf-python/blob/master/setup.py#L25-L45 then it should be specified in `buildInputs = [ ]`.

**Comment by costrouc** [comment]
Please move to `gfortran` to `buildInputs = [ ]` if it is only a build dependency. Otherwise looks great to me!

**Comment by Ekleog** [comment]
@mhaselsteiner Are you still interested in pushing this forward? It should be just a one-line change, but now there is a merge conflict, so it'll maybe be a bit harder.

**Comment by mhaselsteiner** [comment]
I resolved the conflict and added some tests

**Comment by ryantm** [comment]
@GrahamcOfBorg build pythonPackages.wrf-python

**Pull request merged**